_scripts/instruction-widget/templates/install/commonauto.html

```diff
@@ -2,7 +2,9 @@
 Since it doesn't seem like your operating system has a packaged version of Certbot, you should use our certbot-auto script to get a copy:
 </p>
 <pre>wget https://dl.eff.org/certbot-auto
```

Comment:
What are the default file permissions for the file created by `wget`? I think we should verify this file is **always** created with permissions we're OK with (e.g. not world writeable) or change the `chmod` command to `sudo chmod 755 ...`.

Reply:
I'm up to changing the chmod to octal mode. More context in the [comment below](https://github.com/certbot/website/pull/406#discussion_r276008145)

Reply:
Works for me!